doc/cli.markdown

```diff
@@ -1302,9 +1302,9 @@ answer "yes" to all questions (non interactive use)
 
 # Logs
 
-## logs &#60;uuidOrDevice&#62;
+## logs &#60;device&#62;
```

Comment:
I changed the argument name from `uuidOrDevice` to `device` (which I think makes more sense, since it can be one of : device uuid, device IP, device hostname).